### PR TITLE
feat: add upps to update final fees

### PR DESCRIPTION
**Pull request by md0x**
Changes proposed in this PR:
- Add UPP to update final fees to 250$

Fixes UMA-1560

### CODE REVIEW

File: UPPs/upp-14.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Summary
+
+This UPP proposes to update the final fees for various registered stablecoin collateral types in the UMA system. This has been restricted to WETH and stables that are actually used today or have a higher likelihood of usage. These include: WETH, DAI, USDC and USDT.
+
+## Rationale
+
+UMA's final fees serve as a mechanism to prevent spam within the system. The primary objective is to discourage unnecessary or malicious escalation to the DVM (Data Verification Mechanism) without any significant cost. Previously, the final fees were set at $500 when the DVM2.0 was not yet implemented.
+
+With the introduction of DVM 2.0, new anti-spam measures have been implemented, ensuring robust protection against spam and misuse. These improved mechanisms have made it possible to lower the final fees to $250, independent of any changes in the value of $UMA.
```
Comment (daywiss) [inline UPPs/upp-14.md:13]: would be nice to know what the current values are

**Review by daywiss — COMMENTED** [review]
lgtm, but will leave to someone with more context to approve

File: UPPs/upp-14.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Specifics
+
+In order to implement this upgrade, the final fees for all the mentioned currencies should be adjusted to an equivalent of $250 USD worth of tokens each.
+
+**WETH**
+- WETH Mainnet: [0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2](https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2)
+- WETH Polygon: [0x7ceb23fd6bc0add59e62ac25578270cff1b9f619](https://etherscan.io/token/0x7ceb23fd6bc0add59e62ac25578270cff1b9f619)
+- WETH Optimism: [0x4200000000000000000000000000000000000006](https://etherscan.io/token/0x4200000000000000000000000000000000000006)
+- WETH Arbitrum: [0x82af49447d8a07e3bd95bd0d56f35241523fbab1](https://etherscan.io/token/0x82af49447d8a07e3bd95bd0d56f35241523fbab1)
```
Comment (mrice32) [inline UPPs/upp-14.md:25]:
```suggestion
- WETH Polygon: [0x7ceb23fd6bc0add59e62ac25578270cff1b9f619](https://polygonscan.com/token/0x7ceb23fd6bc0add59e62ac25578270cff1b9f619)
- WETH Optimism: [0x4200000000000000000000000000000000000006](https://optimistic.etherscan.io/token/0x4200000000000000000000000000000000000006)
- WETH Arbitrum: [0x82af49447d8a07e3bd95bd0d56f35241523fbab1](https://arbiscan.io/token/0x82af49447d8a07e3bd95bd0d56f35241523fbab1)
```

**Review by mrice32 — APPROVED** [review]
Just a few minor link updates!

**Review by mrice32 — APPROVED** [review]

**Pull request merged**